tests/e2e/vmservice_utils.go

```diff
@@ -32,9 +32,10 @@ import (
 
 	"github.com/onsi/gomega"
 	"github.com/pkg/sftp"
+	"github.com/vmware-tanzu/vm-operator/api/v1alpha2/common"
 	"golang.org/x/crypto/ssh"
 
-	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha1"
+	vmopv2 "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
```

Comment:
Suggestion: keep your import alias as `vmopv1` `cause it is still v1 and the diff is smaller. For whenever we get around to a v1a3, it will be an small, incremental change from v1a2 so most likely the only change required here will be to update the import.

Reply:
Fixed.